OverlayPlugin.Core/ClipboardTechSupport.cs

```diff
@@ -232,6 +281,46 @@ private void GetFFXIVScreenMode(Process process)
             return;
         }
 
+        private void GetFFXIVIsRunningAsAdmin(Process process)
+        {
+            if (process == null)
+            {
+                ffxivIsAdmin = "(not running)";
+                return;
+            }
+
+            try
+            {
+                OpenProcessToken(process.Handle, TOKEN_QUERY, out ph);
+                ffxivWi = new WindowsIdentity(ph);
+                if (ffxivWi.Owner.IsWellKnown(WellKnownSidType.BuiltinAdministratorsSid))
+                {
+                    ffxivIsAdmin = "Elevated (Admin)";
+                }
+                else
+                {
+                    ffxivIsAdmin = "Not Elevated";
+                }
+            }
+            catch (Exception)
+            {
+                // Will get an access-denied exception if ACT is not elevated, but FFXIV is,
+                // since ACT won't have sufficient permissions to check the FFXIV process.
+                // Could theoretically be triggered if FFXIV is running under a different
+                // (non-admin) user, so give a somewhat non-comittal output.
+                // NOTE: *REALLY REALLY* bad idea to run FFXIV as elevated, as it will result
+                // in game config data becoming write-protected from your own user account.
```

Comment:
FFXIV runs as admin whenever it's patched (until restarted), so this comment is a bit disingenuous.

The game's config files should always be readable by the user regardless of elevation state because they reside in the `Documents` folder of the user's profile which has special considerations for permissions for files created in it.

Reply:
I know the game runs as elevated after a patch, but I think the comment as written is still accurate?  If FFXIV is run as an elevated process, any writes to the game config files result in the config files being effectively read-only from that point forward to anything other than an admin user, which would then mean that later running the game as a non-elevated user would prevent changes to the config files.  I seem to remember seeing something about this re: XIVLauncher a while back, but I can't dig it up atm.

Regardless, it is a bit of a gratuitous comment, so I'm fine removing it if it doesn't add value.